modules/kubernetes-addons/crossplane/locals.tf

```diff
@@ -0,0 +1,39 @@
+locals {
+  name                 = "crossplane-system"
+  service_account_name = "crossplane"
```

Comment:
service account name is the same as values on line 5 and 6. Prob can reword vars. Also convention in the codebase is to use the `-sa` postfix for namespaces yea?

Reply:
Crossplane doesn't provide option to create our own SA. I just removed the reference. SA will be created by Helm chart.

We may need to rethink about adding `-sa` service accounts and `-ns` to namespaces. This doesn't look good. We can identify the service accounts using our SSP tags anyway